minCostFlow.py

```diff
@@ -41,10 +45,40 @@ def construct_digraph(edges_file, cap):
             if not node2 in idDict:
                 idDict[node2] = curID
                 curID += 1
-            #Google's solver can only handle int weights, so round to the 100th
-            w = int((1-(float(tokens[2])))*100)
-            G.add_arc_with_capacity_and_unit_cost(idDict[node1],idDict[node2], default_capacity, int(w))
-            G.add_arc_with_capacity_and_unit_cost(idDict[node2],idDict[node1], default_capacity, int(w))
+            # Google's solver can only handle int weights, so round to the 100th 
+            w = int((1-(float(tokens[2])))*100) # lower the weight from token[2], higher the cost
+            d = tokens[3]
```

Comment:
We may want to add some line validation and check we have 4 tokens after splitting the line.